app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  before_action :login_confirm
+
+  #user's homepage controller
+  def homepage
+
+  end
+
+
+
+
+
+  private
+  def login_confirm
+    if !user_signed_in?
+      redirect_to new_user_session_path  # if the user is not signed in redirect him to sign in
+
+    elsif current_user.admin == true
```

Comment:
`current_user.admin` is already returning a boolean value so `== true` part is never required.